lib/Errors.js

```diff
@@ -34,6 +34,11 @@ class UnkownRequestType extends Error {
 
 class OnSessionEndedError extends Error {
   constructor(errorOnSession) {
+    if (errorOnSession instanceof Object && errorOnSession.constructor === Object) {
+      const replacer = null;
+      const space = 2;
+      errorOnSession = JSON.stringify(errorOnSession, replacer, space);
```

Comment:
Do we really need to create a variable for replace and space?
Can we just do `JSON.stringify(errorOnSession, null, 2);`

Reply:
Agree with octavio, i understand those variables are helpful if you don't know what the null and 2 values mean, but in this case JSON.stringify is a common enough function with javascript devs that we don't really need it

Reply:
Ok, I'll delete them then